### PR TITLE
[ADD] partner_contact_height and partner_contact_weight

### PR DESCRIPTION
**Pull request by max3903**
#335

### CODE REVIEW

**Review by susport — COMMENTED** [review]
To correct travis error. Put new line at end of files

File: partner_contact_personal_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#. In the Personal Information tab, enter its height and weight
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/9.0
+
+Known issues / Roadmap
+======================
+
+* ...
```
Comment (damendieta) [inline partner_contact_personal_info/README.rst]: Is this tag need if there is no known issue?

File: partner_contact_personal_info/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Personal Information",
```
Comment (damendieta) [inline partner_contact_personal_info/__openerp__.py]: Maybe a name like Partner Contact Weight and height could be more accurate, as there are several modules that increase personal info like date of birth...

**Comment by pedrobaeza** [comment]
There's already https://github.com/OCA/partner-contact/tree/9.0/partner_contact_personal_information_page. Why a new one?

**Comment by damendieta** [comment]
I agree with @pedrobaeza maybe it's better to include this information in the partner_contact_personal_information_page module.

In the other hand, just a few companies would need this information, maybe a boolean on a configuration page to show/hide fields.

**Comment by max3903** [comment]
@pedrobaeza Because you and I haven't seen it when we made/reviewed the issue #335

**Comment by damendieta** [comment]
Apart of some missing newlines at the end of files, it's ok to me.

**Review by damendieta — APPROVED** [review]

File: partner_contact_height/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (ivantodorovich) [inline partner_contact_height/__init__.py]: New line

File: partner_contact_height/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import res_partner
```
Comment (ivantodorovich) [inline partner_contact_height/models/__init__.py]: New line

File: partner_contact_weight/__init__.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (ivantodorovich) [inline partner_contact_weight/__init__.py]: New line

File: partner_contact_weight/models/__init__.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import res_partner
```
Comment (ivantodorovich) [inline partner_contact_weight/models/__init__.py]: New line

**Review by ivantodorovich — APPROVED** [review]
Just a few new lines missing.
Otherwise, LGTM 👍

**Pull request merged**